web_src/css/base.css

```diff
@@ -435,6 +435,10 @@ a.label,
   box-shadow: 1px 1px 0 0 var(--color-secondary);
 }
 
+.ui.modal .actions {
+  border-radius: 0 0 var(--border-radius) var(--border-radius);
+}
```

Comment:
There is `modal.css` and there are more accurate selectors for `actions`.

Reply:
I made this inprecise intentionally because the original fomantic selector does not match all variants seen on the devtest page.

Reply:
I will take a try, and make devtest page cover all cases.

Reply:
It should be like this. The details were written in the modal's comments.

![image](https://github.com/go-gitea/gitea/assets/2114189/ea94476f-ccb3-41a8-bf33-72911b7b54f7)


![image](https://github.com/go-gitea/gitea/assets/2114189/c136bb68-bbd0-4adc-a8b9-c9bdbd4bddf9)

Reply:
Looks good.